app/views/courses/manage_unlocks.html.erb

```diff
@@ -28,7 +28,23 @@
   };
 </script>
 
-<h1>Edit unlock conditions</h1>
+<h1 class="float-left">Edit unlock conditions</h1>
+
+<div class="float-right">
+  <p>Syntax:</p>
+<pre class="unlock-help">
+exercise [group] &lt;exercise-or-group&gt;
```

Comment:
Maybe split this into two lines?
exercise <exercise>
exercise group <exercise-group>